lib/std/os.zig

```diff
@@ -5465,3 +5465,30 @@ pub fn prctl(option: i32, args: anytype) PrctlError!u31 {
         else => |err| return std.os.unexpectedErrno(err),
     }
 }
+
+pub const GetrlimitError = UnexpectedError;
+
+pub fn getrlimit(resource: rlimit_resource, limits: *rlimit) GetrlimitError!void {
```

Comment:
Precedent in the rest of this file:
```suggestion
pub fn getrlimit(resource: rlimit_resource) GetrlimitError!rlimit {
```